src/styles.html

```diff
@@ -1,8 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <link href='http://fonts.googleapis.com/css?family=Ubuntu:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="styles/main.min.css">
+    <link
+      href="http://fonts.googleapis.com/css?family=Ubuntu:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400..700;1,400..700&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Public+Sans:ital,wght@0,900;1,900&display=swap"
+      rel="stylesheet"
```

Comment:
Noting that my `prettier` setup formatted this file, but all that actually changed is switching from Ubuntu to Open Sans as the preferred font